Skier summary page enhancement

Jar0T opened the pull request:
Displayed data comes from database and added ability to choose specific lift

Jar0T committed:
Skier summary page getting and displaying data from DB

Jar0T committed:
Added ability to choose specific lift in Skier Summary Page

syphzur commented on src/E-SkiLift/Models/SkierSummaryModel.cs:
```diff
@@ -0,0 +1,82 @@
+            TicketID = ticketID;
+
+            SummaryEntryList = new List<SummaryEntry>();
+            for (int i = 0; i < liftUsageHistory.Length; i++)
```
i think it's better to use foreach-in loop to keep code readable

syphzur reviewed (changes requested):
please change for loops to foreach-in

Jar0T committed:
Changed regular for for foreach

Jar0T committed:
Resetting combobox after returning from summary page

Jar0T commented:
Changed for loops to foreach and fixed bug that occured if user selected specific lift and returned from summary page and then wanted to see summary for another ticket that wasn't used at previously selected lift.

syphzur commented:
> Changed for loops to foreach and fixed bug that occured if user selected specific lift and returned from summary page and then wanted to see summary for another ticket that wasn't used at previously selected lift.

mmmm mmmm mmm really nice
